From b0d31bcd5afbf9b63bac3a1731b5364d101631b3 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:08:13 +0200
Subject: [PATCH 2/2] Merge android-9.0.0_r12

commit 2c60d23f7a992de216599e2c6e0e72506049bd15
Merge: 4fed91fe4 0efa4756f
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 8 03:08:47 2018 +0000

    Snap for 4880674 from 0efa4756f2493fbfbe75338659461ceb4701595c to pi-dr1-release

    Change-Id: I72b82fdb2db3a8bcc1dd601fa96905e6b411b12a

commit 0efa4756f2493fbfbe75338659461ceb4701595c
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 5 19:52:11 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ica98d24186da935032b908019c91fa0b938465a0

commit 4fed91fe4b0c526c86e0e001e2c7b3ecb3cd0443
Merge: 3197fd8be f9bc66fd2
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:04:09 2018 +0000

    Snap for 4820872 from f9bc66fd21a9a19418914ebb8feba5797e138358 to pi-dr1-release

    Change-Id: I2d1d63095c87755e7820a01e824b9c1beeada8ae

Change-Id: I8c0d368e4f2ef586186c75d61795304f7bfdefef
---
 res/values-sw/strings.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 7a161fa5f..2c27f0207 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -23,12 +23,12 @@
     <string name="alarm_vibrate" msgid="3476686921490362230">"Mtetemo"</string>
     <string name="alarm_repeat" msgid="7242985466344233206">"Rudia"</string>
     <string name="delete" msgid="5732434972457000541">"Futa"</string>
-    <string name="default_alarm_ringtone_title" msgid="2916376768509623643">"Sauti chaguo msingi ya kengele"</string>
+    <string name="default_alarm_ringtone_title" msgid="2916376768509623643">"Sauti chaguomsingi ya kengele"</string>
     <string name="alarm_sound" msgid="8754499591391638405">"Mlio wa kengele"</string>
     <string name="timer_sound" msgid="896163086604894735">"Mlio wa kipima muda"</string>
     <string name="add_new_sound" msgid="2546936797351540244">"Ongeza mpya"</string>
     <string name="remove_sound" msgid="2064643204455734239">"Ondoa"</string>
-    <string name="confirm_remove_custom_ringtone" msgid="1724111242265208553">"Kengele na vipima muda vinavyotumia sauti hii vitacheza sauti chaguo msingi badala yake."</string>
+    <string name="confirm_remove_custom_ringtone" msgid="1724111242265208553">"Kengele na vipima muda vinavyotumia sauti hii vitacheza sauti chaguomsingi badala yake."</string>
     <string name="your_sounds" msgid="1672779351352629681">"Sauti zako"</string>
     <string name="device_sounds" msgid="1584894148222639823">"Sauti za kifaa"</string>
     <string name="custom_ringtone_lost_permissions" msgid="3406310054255354331">"Maudhui ya sauti hayawezi kufikiwa."</string>
@@ -180,7 +180,7 @@
     <string name="unknown_ringtone_title" msgid="6637522029614550112">"Isiyojulikana"</string>
     <string name="alarm_volume_muted" msgid="6658289289695673569">"Sauti ya kengele imezimwa"</string>
     <string name="unmute_alarm_volume" msgid="2655009276035357497">"Iwashe"</string>
-    <string name="silent_default_alarm_ringtone" msgid="6012855475009670997">"Mlio chaguo msingi wa kengele ni kimya"</string>
+    <string name="silent_default_alarm_ringtone" msgid="6012855475009670997">"Mlio chaguomsingi wa kengele ni kimya"</string>
     <string name="change_setting_action" msgid="1224356633832430160">"Badilisha"</string>
     <string name="app_notifications_blocked" msgid="7249150789023936889">"Arifa za saa zimezuiwa"</string>
     <string name="alarms_blocked_by_dnd" msgid="6089433757505898969">"Kifaa kimewekwa katika hali ya kimya"</string>
-- 
2.17.1

